model/src/main/java/org/perfrepo/dto/test/TestSearchParams.java

```diff
@@ -0,0 +1,131 @@
+package org.perfrepo.dto.test;
+
+import java.util.List;
+
+/**
+ * Data transfer object that contains filter parameters for {@link TestDto} searching query.
+ *
+ * @author Jiri Grunwald (grunwjir@gmail.com)
+ */
+public class TestSearchParams {
+
+    private List<String> nameFilters;
+
+    private List<String> uidFilters;
+
+    private List<Integer> groupIdFilters;
```

Comment:
I guess you want to pass Strings instead of Integer, since user will surely specify the group by it's name.